SwiftWisdom/Core/StandardLibrary/String/String+Indexing.swift

```diff
@@ -14,6 +14,24 @@ extension String {
         return String(substringCharacters)
     }
 
+    /// Returns a substring in the given range.
+    /// If the range is beyond the string's length, returns the original string.
```

Comment:
Oops, yes, I actually had it this way and then changed it. Now I remember why I did it like that! Good catch.